.github/actions/prepare-build-env/action.yml

```diff
@@ -2,11 +2,20 @@ name: Prepare build environment (Linux only)
 description: Set up Java, Maven, Gradle, etc.
 runs:
   using: "composite"
-  steps: 
-    - name: Set up Java JDK
-      uses: actions/setup-java@v3.3.0
-      with:
-        java-version: 17
-        distribution: zulu      
+  steps:
+    - name: Set up Java 17
+      run: |
+        echo "$JAVA_HOME_17_X64/bin" >> $GITHUB_PATH
+        echo "org.gradle.java.home=${JAVA_HOME_17_X64//\\/\/}" >> gradle.properties
```

Comment:
What's the `//\\/\/` for?

Reply:
On Windows, the default path separator `\`needs to be replaced with the Unix-style `/`.